README.md

```diff
@@ -371,6 +376,87 @@ Failed: text 'foo' not present on element with locator 'By(css selector, h1)'.
 
 > As you can see, the messages are clear and tell you exactly why the test has failed, such as in the previous example, where a specific text ('foo') is not present in a heading element (an element with css selector 'h1').
 
+## Preparation to next major version
+
+We will show some alerts on console to the user know what changes he need to do.
+
+All changes below is for the library focus only on providing an easy way for software engineers to create robust end-to-end tests.
```

Comment:
What about: All the below changes intend to make it easier for software engineers to create robust end-to-end tests. ?